src/d4bl/observability/langfuse.py

```
@@ -5,17 +5,47 @@
+        if ":3002" in host:
+            host = host.replace(":3002", ":3000")
+    return host
 
-def initialize_langfuse():
+
+def check_langfuse_service_available(host: str, timeout: float = 3.0) -> bool:
+    """Check if Langfuse service is reachable via HTTP GET."""
+    import urllib.request
+
+    try:
+        resp = urllib.request.urlopen(f"{host}/api/public/health", timeout=timeout)
```

Ruff (0.15.2)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)